### ISSUE TITLE
Better explanations for fields

### ISSUE THREAD

**Issue by juliusdanek** [sorare/api#133]
It would be really really great if you guys could add more detailed explanations to API fields. A lot are self-explanatory but there's a large percentage where you are left scratching your head. Here's one for example:

<img width="577" alt="Screenshot 2022-02-23 at 14 56 40" src="https://user-images.githubusercontent.com/9576231/155407008-c9bca121-06a6-49e3-a790-3187840e9d57.png">

**Comment by redox** [comment]
Agreed, we have done a full pass a couple of month ago and added a bunch of description; but some are still missing. Also, the default TYPE DETAILS isn't helping :)

**Comment by juliusdanek** [comment]
Yup, not so helpful. I do know what a Boolean is! 😂
